src/controllers/tasks/statement.of.capital.controller.ts

```diff
@@ -22,6 +22,7 @@ export const get = async(req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const statementOfCapital: StatementOfCapital = await getStatementOfCapitalData(session, companyNumber);
     const sharesValidation = await validateTotalNumberOfShares(session, companyNumber, +statementOfCapital.totalNumberOfShares);
+    const totalAmountUnpaidValidation = statementOfCapital.totalAmountUnpaidForCurrency !== null;
```

Comment:
statementOfCapital.totalAmountUnpaidForCurrency can also be undefined (more likely than null). To check if it is present try using       typeof statementOfCapital.totalAmountUnpaidForCurrency === 'string' instead